examples/data/json-units/benchmark_production_OECM.json

```diff
@@ -10,6 +10,7 @@
         "benchmark_metric": "dimensionless",
         "scenario name": "OECM 1.5 Degrees",
         "release date": "2022",
+        "base_year_production": "1869.6 Fe * megametric_ton",
```

Comment:
why do we suddenly need this field?
looks like we managed to do our calculations w/o it

Reply:
It's needed for co2budget.ipynb